test.js

```diff
@@ -135,4 +135,29 @@ describe('notify(opts)', () => {
 			''
 		].join('\n'));
 	});
+
+	it('should exclude -g argument when `isGlobal` option is `false`', () => {
+		function Control() {
+			this.packageName = 'update-notifier-tester';
+			this.update = {
+				current: '0.0.2',
+				latest: '1.0.0'
+			};
+		}
+		util.inherits(Control, updateNotifier.UpdateNotifier);
+		const notifier = new Control();
+		notifier.notify({defer: false, isGlobal: false});
+		assert.equal(stripAnsi(errorLogs), [
+			'',
+			'',
+			'   ╭────────────────────────────────────────────────╮',
+			'   │                                                │',
+			'   │        Update available 0.0.2 → 1.0.0          │',
+			'   │   Run npm i update-notifier-tester to update   │',
+			'   │                                                │',
+			'   ╰────────────────────────────────────────────────╯',
+			'',
+			''
+		].join('\n'));
```

Comment:
No need to test the whole output. Having many of these will make it hard to change for example the box style later. Instead just do a indexOf check for `Run npm i update-notifier-tester to update`, after running it through stripAnsi of course.